types/syslogfacility.pp

```diff
@@ -0,0 +1,25 @@
+type Stdlib::Syslogfacility = Enum[
+  'user',
```

Comment:
There's also a `kern` facility.  I can't remember why I left it out of the nrpe data type.  Probably because it wouldn't make sense (or be illegal) to log nrpe with facility code 0 (not that many of the others would make much sense either).

Reply:
updated it